src/Libraries/Microsoft.Extensions.AI.Evaluation/NumericMetric.cs

```diff
@@ -24,5 +24,6 @@ namespace Microsoft.Extensions.AI.Evaluation;
 /// An optional string that can be used to provide some commentary around the result represented by this
 /// <see cref="NumericMetric"/>.
 /// </param>
+/// <related type="Article" href="/dotnet/ai/quickstarts/evaluate-ai-response">Evaluate a model's response</related>
```

Comment:
Should we add this to the other metrics too (like StringMetric and BooleanMetric) or perhaps we can remove it from NumericMetric and add this on the base EvaluationMetric type)?